tests/sqlparser_postgres.rs

```diff
@@ -3199,7 +3199,7 @@ fn parse_drop_mirror() {
 #[test]
 fn parse_mirror_for_select() {
     match pg().verified_stmt("CREATE MIRROR IF NOT EXISTS test_mirror FROM p1 TO p2 FOR $$SELECT 1$$ WITH (key1 = 'value1', key2 = 'value2')") {
-         Statement::CreateMirror { if_not_exists,create_mirror: MirrorSelect(select) } => {
+        Statement::CreateMirror { if_not_exists,create_mirror: MirrorSelect(select) } => {
```

Comment:
Incorrect indentation not being picked up by rustfmt implies some rustfmt.toml should be created since there should be some config to have it notice that